script/testing/util/test_server.py

```diff
@@ -36,6 +38,10 @@ def __init__(self, args):
         self.continue_on_error = self.args.get(
             "continue_on_error", constants.DEFAULT_CONTINUE_ON_ERROR)
 
+        # memory info collection
+        self.collect_mem_info = self.args.get("collect_mem_info", False)
+        self.collect_mem_freq = self.args.get("collect_mem_freq",
```

Comment:
What if we used the `buckets` attribute from the OLTPBench and brought that into the TestServer?

Reply:
This is tricky, because this is `test_server.py` not `test_oltpbench.py`, other tests will also use this part of code, and junit, for example, does not have this `buckets`. So I think maybe it is better to separate those 2 concepts.

Reply:
Yah I'm suggesting moving buckets into the test_server because it now has meaning everywhere. It would mean the memory collection interval (maybe we rename buckets) then we can remove that OLTPBench specific part and use the test_server buckets

Reply:
Hmm I dont think so. Because we have 3 tests now and only OLTPBench has bucket concept, JUnit and MicroBench do not even have this concept nor have any incremental metrics. I prefer to keep it the current way to keep the bucket concept within the OLTPBench.

Reply:
replaced it by `incremental_metric_freq`